jme3-lwjgl3/src/main/java/com/jme3/system/lwjgl/LwjglWindow.java

```diff
@@ -281,11 +282,16 @@ public void invoke(int error, long description) {
 
         final GLFWVidMode videoMode = glfwGetVideoMode(glfwGetPrimaryMonitor());
 
-        if (settings.getWidth() <= 0 || settings.getHeight() <= 0) {
-            settings.setResolution(videoMode.width(), videoMode.height());
+        if (settings.getWindowWidth() <= 0 || settings.getWindowHeight() <= 0) {
+            settings.setWindowSize(videoMode.width(), videoMode.height());
```

Comment:
Please check whether `videoMode` is null from an error occurring during `glfwGetVideoMode()`.

Reply:
This issue is not related to this PR and might need some error handling. It is better to fix it with a commit later